experimenter/manifesttool/cli.py

```diff
@@ -42,112 +39,16 @@ def fetch_latest(ctx: click.Context):
     """Fetch the latest FML manifests and generate experimenter.yaml files."""
     context = ctx.find_object(Context)
 
+    results = []
+
     for app_name, app_config in context.app_configs.__root__.items():
         context.manifest_dir.joinpath(app_config.slug).mkdir(exist_ok=True)
 
         if app_config.fml_path is not None:
-            fetch_fml_app(context, app_name, app_config)
+            results.append(fetch_fml_app(context.manifest_dir, app_name, app_config))
         elif app_config.experimenter_yaml_path is not None:
-            fetch_legacy_app(context, app_name, app_config)
+            results.append(fetch_legacy_app(context.manifest_dir, app_name, app_config))
         else:  # pragma: no cover
             assert False, "unreachable"
```

Comment:
Wait why is this here?

Reply:
We don't actually enforce that one of fml_path or experimenter_yaml_path is non none but in practice there is one or the other. Ill follow up to make it required in the model